src/googmodule.ts

```diff
@@ -0,0 +1,388 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import * as path from 'path';
+
+import {ModulesManifest} from './modules_manifest';
+import {visitNodeWithSynthesizedComments} from './transformer_util';
+import * as ts from './typescript';
+
+export interface GoogModuleProcessorHost {
+  /**
+   * Takes a context (the current file) and the path of the file to import
```

Comment:
The current file... path?  Relative to what?

Reply:
Done.